xbmc/rendering/RenderSystem.h

```diff
@@ -39,6 +40,7 @@ class CRenderSystemBase
   virtual void PresentRender(bool rendered, bool videoLayer) = 0;
   virtual bool ClearBuffers(UTILS::COLOR::Color color) = 0;
   virtual bool IsExtSupported(const char* extension) const = 0;
+  virtual bool IsExtSupported(const GLEXTENSIONS::EXTENSION extension) { return false; }
```

Comment:
This somewhat breaks abstraction by putting derived class specifics into the base class. I would think extensions would exist for the base class.

It is possible to get rid of this and cast winSystem where needed.

CRenderSystemBaseand CWindowSystemBase seem to be a dumping ground for non abstract methods.